stacker_blueprints/empire/policies.py

```diff
@@ -116,12 +149,23 @@ def empire_policy(resources):
                 Effect=Allow,
                 # TODO: Limit to specific ELB?
                 Resource=["*"],
-                Action=[elb.DeleteLoadBalancer, elb.CreateLoadBalancer,
-                        elb.DescribeLoadBalancers, elb.DescribeTags,
-                        elb.ConfigureHealthCheck,
-                        elb.ModifyLoadBalancerAttributes,
-                        elb.SetLoadBalancerListenerSSLCertificate,
-                        elb.SetLoadBalancerPoliciesOfListener]),
+                Action=[
+                    elb.Action("Describe*"),
+                    elb.CreateLoadBalancer,
```

Comment:
If using SSL, i think you'll need `elb.AddTags` as well. My deployment to external, SSL app failed due to this permission. Patched it then deploy worked.